main.go

```diff
@@ -226,6 +300,27 @@ func monitorNetworkState(testNetwork *network.Network) (awResultsWriters []*csv.
 		}))
 	}
 
+	// Here we only monitor the opinion weight of node w/ the highest weight
+	dsPeer := testNetwork.Peers[0]
+	dsPeer.Node.(*multiverse.Node).Tangle.OpinionManager.Events.ApprovalWeightUpdated.Attach(events.NewClosure(func(opinion multiverse.Color, delta_weight int64) {
+		opinionWeightMutex.Lock()
+		defer opinionWeightMutex.Unlock()
+
+		opinionsWeights[opinion] += delta_weight
+	}))
```

Comment:
Should we monitor it on a node with highest mana or the one with lowest mana? 
I also added counters that track how many nodes in the whole network have confirmed each color.

Reply:
1) I think we can let which node to be monitored defined by the user, so might need an extra parameter.
2) Cool. Here the opinion weights are updated before the new opinion formed, so should be different purposes.